### PR TITLE
Fix xrates query

### PR DESCRIPTION
**Pull request by LefterisJP**
#### Fix xrates scraping by introducing requests_html

requests_html is a the most lightweight package I could find that does
scraping after succesfully rendering javascript

#### Move requests_html HTMLSession to a global object

It's a bit ugly but needed since initializing and killing the session
every time a request is made seems to re-create the browser instance
that renders javascript and this is a bit too much to do in every request.

### CODE REVIEW

File: rotkehlchen/server.py [hunk 1]
```diff
@@ -32,6 +33,7 @@ def __init__(self) -> None:
         self.rotkehlchen = Rotkehlchen(self.args)
         self.stop_event = gevent.event.Event()
         domain_list = []
+        GlobalRequestsHTML()  # initialize the llobal requests_html session
```
Comment (LefterisJP) [inline rotkehlchen/server.py:36]:
```suggestion
        GlobalRequestsHTML()  # initialize the global requests_html session
```

aah ffs lefteri

**Review by yabirgb — COMMENTED** [review]
Thanks. Looks okey. Only found a typo

File: rotkehlchen/server.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (LefterisJP) [reply rotkehlchen/server.py:36]: as you can see from my own review I also saw it earlier :smile:

**Comment by LefterisJP** [comment]
Maybe let's not merge this PR even if all tests pass.

The xrates test started passing again. I have no idea why.

**Comment by LefterisJP** [comment]
I will keep this PR open as an experiment as we are experimenting with asyncio-gevent and a javascript rendering engine for parsing, but since the original reason this started (xrates parsing) seems to have been a non-issue and still works in develop, this will not be merged -- and may never be.

Just keeping it as an experiment/testbed here

**Comment by LefterisJP** [comment]
closing since this won't get merged and can always check history by looking for the PR

**Pull request closed without merging**